documentation/tutorials/get-started.md

```diff
@@ -92,7 +92,7 @@ Add `:ash` to your `.formatter.exs` file
 ```elixir
 # Used by "mix format"
 [
-  import_deps: [:ash], # <-- add this line, if you have more import_deps, just add it within the array
+  import_deps: [:ash], # <-- add this line, if you have more import_deps, just add it within the list
```

Comment:
It might seem pedantic and nit-picky. But It made me pause for a moment while reading the tutorial.